applications/mp4box/fileimport.c

```diff
@@ -1955,7 +1955,7 @@ GF_Err cat_isomedia_file(GF_ISOFile *dest, char *fileName, u32 import_flags, GF_
 
 					segmentDuration += (u64) (s64) dur;
 					gf_isom_modify_edit(dest, dst_tk, j, segmentDuration, mediaTime, editMode);
-					break;
+					continue;
 				}
 			}
 		} else {
```

Comment:
why are you changing the break to continue ? That code is supposed to change only one edit list entry

Reply:
I misread this, I thought we were iterating on the tracks, not the edits. Reverting.